dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/ServletFilter.java

```diff
@@ -0,0 +1,273 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.http.monitor;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.ws.webcontainer.srt.SRTServletRequest;
+import com.ibm.ws.webcontainer40.osgi.webapp.WebAppDispatcherContext40;
+import com.ibm.wsspi.webcontainer.webapp.IWebAppDispatcherContext;
+
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletMapping;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+/**
+ *
+ */
+public class ServletFilter implements Filter {
+	
+	private static final TraceComponent tc = Tr.register(ServletFilter.class);
+	
+	private static final String REST_HTP_ROUTE_ATTR = "RESTFUL.HTTP.ROUTE";
```

Comment:
`REST_HTTP_ROUTE_ATTR` or `RESTFUL_HTTP_ROUTE_ATTR`?

Reply:
I guess the value should be  REST.HTTP.ROUTE